src/vuestic-theme/vuestic-components/vuestic-simple-select/VuesticSimpleSelect.vue

```diff
@@ -57,18 +68,25 @@
       name: {
         type: String,
         default: 'simple-select'
+      },
+      isShown: {
```

Comment:
Why do you need that?

Reply:
to check if use select with input, or without